### PR TITLE
Fixed Issue

### PR DESCRIPTION
**Pull request by cloudbehl**
Tendrl-bug-id: #186, #166, #164

### CODE REVIEW

**Comment by cloudbehl** [comment]
@rishubhjain Please review

File: etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-hosts.json [hunk 1]
```diff
@@ -213,15 +215,16 @@
...
                 "gauge": {
                     "maxValue": 100,
                     "minValue": 0,
-                    "show": true,
+                    "show": false,
                     "thresholdLabels": false,
                     "thresholdMarkers": true
                 },
-                "id": 16,
+                "id": 34,
```
Comment (rishubhjain) [inline etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-hosts.json]: Why is this required?

Comment (rishubhjain) [reply etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-hosts.json]: @cloudbehl

Comment (cloudbehl) [reply etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-hosts.json]: @rishubhjain Panel Id, To uniquely identify panels.

**Comment by rishubhjain** [comment]
@r0h4n Please review.
